assets/models/system/microsoft-orca-2-13b/description.md

```diff
@@ -0,0 +1,99 @@
+{
+  "output": [
+    "<|im_start|>system You are Orca, an AI language model created by Microsoft. You are a cautious assistant. You carefully follow instructions. You are helpful and harmless and you follow ethical guidelines and promote positive behavior.<|im_end|> <|im_start|>user How can you determine if a restaurant is popular among locals or mainly attracts tourists, and why might this information be useful?<|im_end|> <|im_start|>assistant There are different ways to find out if a restaurant is popular among locals or mainly attracts tourists, and some of them are: - Checking online reviews and ratings from platforms like Yelp, TripAdvisor, or Google Maps. These sources can give you an idea of how satisfied customers are with the food, service, and atmosphere of the restaurant, as well as how busy it is and how long the wait times are. - Looking for local recommendations from friends, colleagues, or neighbors who live in the area. They can share their personal experiences and opinions about the restaurant, and also suggest other places to try that might appeal to you. - Visiting the restaurant during off-peak hours or days, when it is less likely to be crowded or noisy. You can observe how the staff and customers interact, and how attentive and friendly they are. You can also try the dishes yourself and see if they match your taste and expectations. - Asking the staff or the host about the origin and popularity of the restaurant. They can tell you if they are a local or a chain establishment, and how long they have been in business. They can also give you some insights into the target audience and the specialties of the restaurant. Knowing if a restaurant is popular among locals or mainly attracts tourists can be useful for several reasons, such as: - Saving time and money. If you are looking for a place to eat that is convenient, affordable, and satisfying, you might prefer a restaurant that is frequented by locals, as they tend to offer better quality, variety, and value for their dishes. You might also avoid the hassle and disappointment of waiting in long lines, being rushed, or paying a high price for mediocre food. - Enhancing your cultural and social experience. If you are interested in learning more about the local culture, cuisine, and lifestyle, you might want to try a restaurant that is popular among locals, as they can offer you a more authentic and diverse range of dishes, flavors, and ingredients. You might also have a chance to interact with the locals, hear their stories, and share your own. - Avoiding tourist traps and scams. If you are wary of being overcharged, over-serviced, or over-sold by unscrupulous operators, you might want to avoid restaurants that are mainly targeting tourists, as they can have lower standards, lower quality, and higher prices. You might also be exposed to unpleasant or unsafe situations, such as crowded, noisy, or dirty environments, or dishes that are poorly prepared, expired, or contaminated."
```

Comment:
the output also contains these tokens. I am guessing we should not include input text for this scenario. Use "return_full_text": false

Reply:
in the params?

Reply:
Yes, in the parms to prevent input from coming in the output, especially with start end tokens. Otherwise its ok. Non blocker though. Feel free to merge and update in future